lightning_pose/models/backbones/torchvision.py

```diff
@@ -112,6 +113,10 @@ def build_backbone(
         # load resnet or efficientnet models from torchvision.models
         base = getattr(tvmodels, backbone_arch)(weights=weights)
 
+        if not pretrained and backbone_arch == "resnet50":
```

Comment:
i thought we agreed to not proceed this way - we want to keep the backbone intact, and not change the number of channels in the first layer. instead, take a batch of shape (batch, n_view, 3, height, width), reshape to (batch * n_views, 3, height, width), push through the network, then reshape back to (bach, n_view, n_keypoints, height, width)